src/postgres/src/test/regress/yb_pg_foreign_data_schedule

```diff
@@ -0,0 +1,10 @@
+# src/test/regress/yb_pg_foreign_data_schedule
+#
+####################################################################################################
+# Postgres Testsuites: Porting from PostgreSQL original tests.
+# This includes tests related to postgres foreign data.
+####################################################################################################
+# Dependency for `yb_pg_foreign_data`
+test: yb_create_function
```

Comment:
Hey @radekg, currently `TestPgRegressTypesUDT` is failing because the test executes `yb_pg_create_function_1.source` and `yb_create_function` - both of which now create the test_fdw_handler function and so the second `CREATE FUNCTION` throws an error. 
We can add `yb_pg_create_function_1` to the schedule here instead